src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -13,245 +13,462 @@ public enum SyntaxKind : ushort
         // additional preprocessor keywords
+        /// <summary>Represents <see langword="elif"/>.</summary>
         ElifKeyword = 8467,
+        /// <summary>Represents <see langword="endif"/>.</summary>
         EndIfKeyword = 8468,
+        /// <summary>Represents <see langword="region"/>.</summary>
         RegionKeyword = 8469,
+        /// <summary>Represents <see langword="endregion"/>.</summary>
         EndRegionKeyword = 8470,
+        /// <summary>Represents <see langword="define"/>.</summary>
         DefineKeyword = 8471,
+        /// <summary>Represents <see langword="undef"/>.</summary>
         UndefKeyword = 8472,
+        /// <summary>Represents <see langword="warning"/>.</summary>
         WarningKeyword = 8473,
+        /// <summary>Represents <see langword="error"/>.</summary>
         ErrorKeyword = 8474,
+        /// <summary>Represents <see langword="line"/>.</summary>
         LineKeyword = 8475,
+        /// <summary>Represents <see langword="pragma"/>.</summary>
         PragmaKeyword = 8476,
+        /// <summary>Represents <see langword="hidden"/>.</summary>
         HiddenKeyword = 8477,
+        /// <summary>Represents <see langword="checksum"/>.</summary>
         ChecksumKeyword = 8478,
+        /// <summary>Represents <see langword="disable"/>.</summary>
         DisableKeyword = 8479,
+        /// <summary>Represents <see langword="restore"/>.</summary>
         RestoreKeyword = 8480,
+        /// <summary>Represents <see langword="r"/>.</summary>
         ReferenceKeyword = 8481,
 
+        /// <summary>Represents <c>$"</c> token.</summary>
         InterpolatedStringStartToken = 8482,            // $"
+        /// <summary>Represents <c>"</c> token that is closing <c>$"</c>.</summary>
         InterpolatedStringEndToken = 8483,              // "
+        /// <summary>Represents <c>$@</c> or <c>@$</c> token.</summary>
         InterpolatedVerbatimStringStartToken = 8484,    // $@" or @$"
 
         // additional preprocessor keywords (continued)
+        /// <summary>Represents <see langword="load"/>.</summary>
         LoadKeyword = 8485,
+        /// <summary>Represents <see langword="nullable"/>.</summary>
         NullableKeyword = 8486,
+        /// <summary>Represents <see langword="enable"/>.</summary>
         EnableKeyword = 8487,
 
         // targets for #nullable directive
+        /// <summary>Represents <see langword="warnings"/>.</summary>
         WarningsKeyword = 8488,
+        /// <summary>Represents <see langword="annotations"/>.</summary>
         AnnotationsKeyword = 8489,
 
         // Other
+        /// <summary>Represents <see langword="var"/>.</summary>
         VarKeyword = 8490,
+        /// <summary>Represents <c>_</c> token.</summary>
         UnderscoreToken = 8491,
+        /// <summary>Represents <c>TODO: What's this?</c> token.</summary>
         OmittedTypeArgumentToken = 8492,
```

Comment:
this happens in `typeof(Dictionary<,>)`.  In between the `<,>` are omitted type args.